src/components/Attachment/index.ts

```diff
@@ -579,7 +579,7 @@ class Attachment extends BridgeBase {
     for (const file of arFiles) {
       if (!this.validateMaxSize(file, 5)) {
         file.error = true;
-        file.meta = 'File is too big. Max allowed size is 5MB.';
+        file.meta = this.thePConn.getLocalizedValue('File is too big. Max allowed size is 5MB.', '', '');
```

Comment:
Didn't see a test for this scenario

Reply:
we need 5MB file to test this use case, which we can't ship with our SDK.

Reply:
As long as a manual test has been done.